### DISCUSSION TITLE
请问一下，使用database插件，创建连接的时候找不到html，这种要怎么解决呢，自己新建一个也不管用

### DISCUSSION THREAD

**Discussion by bytemain** [opensumi/core#1697] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/13938334/191434592-b2926ed5-8a0e-4c95-b90f-f356e86dd854.png)

![image](https://user-images.githubusercontent.com/13938334/191434615-a3d2b5ba-a299-409e-a3aa-350b84cf7b59.png)

这个插件创建连接的时候是加载了一个iframe

**Comment by bytemain** [comment]
需要一些更详细的信息：

1. 浏览器可以访问吗？
2. 有可能是插件不兼容 OpenSumi，可以再截图一下 Console 的报错吗
3. 插件名可以发一下，我们来做一下测试

**Comment by hacke2** [comment]
我看下这个问题吧，之前是好的

**Comment by hqiaozhang** [comment]
插件名称是cweijan.vscode-database-client2-5.3.7，浏览器版本是Chrome 97.0.4692.99

**Reply by hacke2** [reply]
这个插件源代码和release 版本都对应不上，确认是正经代码吗？

**Comment by hacke2** [comment]
先临时用下这个版本的插件 [vscode-mysql-client2@3.0.2](https://marketplace.visualstudio.com/_apis/public/gallery/publishers/cweijan/vsextensions/vscode-mysql-client2/3.0.2/vspackage)

**Reply by hqiaozhang** [reply]
还是报错找不到  http://localhost:8899/webview/index.html?id=0 404 (Not Found)

**Reply by hqiaozhang** [reply]
这个index.html需要自己创建一个吗

**Comment by Aaaaash** [comment, marked as answer]
webview 有编译吗，这里的 404 看起来是指加载 webview/index.html 时找不到资源，你可以参考 https://github.com/opensumi/ide-startup/blob/master/configs/webpack.webview.config.js 将 webview 编译后 serve 到 8899 端口，例如使用 webpack-dev-server https://github.com/opensumi/ide-startup/blob/master/package.json#L21

**Reply by hqiaozhang** [reply]
可以了，谢谢
